.pre-commit-config.yaml

```diff
@@ -1,50 +1,36 @@
----
 repos:
-  - repo: meta
-    hooks:
-      # - id: check-hooks-apply
-      - id: check-useless-excludes
-
-  - repo: local
-    hooks:
-      - id: go-version
-        name: go version
-        entry: scripts/check-go-version
-        language: script
-        types: [go]
-
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.3.0
     hooks:
       - id: check-json
       - id: check-merge-conflict
-      - id: check-symlinks
       - id: check-yaml
       - id: detect-private-key
-      - id: end-of-file-fixer
       - id: pretty-format-json
         args:
           - --autofix
-      - id: mixed-line-ending
       - id: trailing-whitespace
 
-  - repo: https://github.com/executablebooks/mdformat
-    rev: 0.7.16
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.32.2
```

Comment:
Admittedly I just copied over `.pre-commit-config.yml` from another repo. One I had updated previously. Though looking at `mdformat`, it does seem useful as I believe it could address some alerts we see with `markdownlint-cli`. I'll fix. Thanks for catching this!